src/coreComponents/dataRepository/MappedVector.hpp

```diff
@@ -489,42 +490,27 @@ T * MappedVector< T, T_PTR, KEY_TYPE, INDEX_TYPE >::insert( KEY_TYPE const & key
     m_constValues.emplace_back( keyName, rawPtr( index ) );
 
   }
-  // if key was found
-  else
+  else // key was found. already exists!
   {
     index = iterKeyLookup->second;
 
-    if( takeOwnership )
-    {
-      m_ownsValues[index] = true;
-    }
-
     // if value is empty, then move source into value slot
     if( m_values[index].second==nullptr )
```

Comment:
I'm curious, when do we use this functionality of inserting a `nullptr`? It seems like the kind of thing that could reasonably be disallowed.

Reply:
....err....good point.